DxDispatch/src/model/Model.cpp

```diff
@@ -36,6 +36,19 @@ Model::Model(
                 }
             }
         }
+        else if (std::holds_alternative<DmlSerializedGraphDispatchableDesc>(dispatchableDesc.value))
+        {
+            auto& dmlSerializedGraphDispatchableDesc = std::get<DmlSerializedGraphDispatchableDesc>(dispatchableDesc.value);
+            for (auto& sourceResource : dmlSerializedGraphDispatchableDesc.initBindings)
+            {
+                if (m_resourceDescsByName.find(sourceResource.first) == m_resourceDescsByName.end())
```

Comment:
initBindings is a `std::unordered_map<std::string, std::vector<BufferBindingSource>>`. We need to iterator on the value of each entry of this map. `sourceResource.first` will give you the name of tensor of a node or model which might be different from the resource name itself. See `dml_owned_tensors.json` model for example.
The logic to check for resource existence should be same as DmlDispatchable above and can be in the same if block.

Reply:
done